pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -511,6 +511,64 @@ downloadPhase:
 	}
 }
 
+// runPlaybook implements the playbook for local scoped app install
+func (ctx *localScopeInstallContext) runPlaybook() {
+	worker := ctx.worker
+	cr := worker.cr
+	scopedLog := log.WithName("playbookForLocalScopeInstall").WithValues("name", cr.GetName(), "namespace", cr.GetNamespace(), "pod", worker.targetPodName, "app name", worker.appDeployInfo.AppName)
+
+	defer func() {
+		// ToDo sgontla: Open the semlock once CSPL-1449 is available
+		//<-ctx.sem
+		worker.isActive = false
+		worker.waiter.Done()
+	}()
+
+	appPkgFileName := getAppPackageName(worker)
+	appPkgPathOnPod := filepath.Join(appBktMnt, worker.appSrcName, appPkgFileName)
+
+	phaseInfo := getPhaseInfoByPhaseType(worker, enterpriseApi.PhaseInstall)
+
+	if !checkIfFileExistsOnPod(*worker.client, cr.GetNamespace(), worker.targetPodName, appPkgPathOnPod) {
+		scopedLog.Error(nil, "app pkg missing on Pod", "app pkg path", appPkgPathOnPod)
+		phaseInfo.Status = enterpriseApi.AppPkgMissingOnPodError
+		return
+	}
+
+	var command string
+	if worker.appDeployInfo.IsUpdate {
+		// App was already installed, update scenario
+		command = fmt.Sprintf("/opt/splunk/bin/splunk install app %s -update 1 -auth admin:`cat /mnt/splunk-secrets/password`", appPkgPathOnPod)
+	} else {
+		command = fmt.Sprintf("/opt/splunk/bin/splunk install app %s -auth admin:`cat /mnt/splunk-secrets/password`", appPkgPathOnPod)
+	}
+
+	streamOptions := &remotecommand.StreamOptions{
+		Stdin: strings.NewReader(command),
+	}
+
+	stdOut, stdErr, err := splutil.PodExecCommand(*worker.client, worker.targetPodName, cr.GetNamespace(), []string{"/bin/sh"}, streamOptions, false, false)
+	if err != nil {
```

Comment:
you should also check `stdErr` value too, since the command can succeed but application can throw some error which is recorded in stdErr